content/community.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Community
+layout: page
+
+---
+# Project Governance
+
+The Prometheus project follows the [Prometheus governance](/governance/).
+
+# Community
+
+Prometheus is developed in the open. Here are some of the channels we
+use to communicate and contribute:
+
+## Slack channel
+
+`#prometheus` on CNCF [Slack](https://slack.cncf.io/).
+
+## IRC
+
+`#prometheus` on [irc.libera.chat](https://libera.chat/). This
+channel is bridged to the Matrix room below.
+
+## Matrix
+
+[`#prometheus:matrix.org`](https://app.element.io/#/room/#prometheus:matrix.org).
+This room is bridged to the IRC room above.
+
+## User mailing lists
+
+-   [prometheus-announce](https://groups.google.com/forum/#!forum/prometheus-announce)
+    ([mirror](https://www.mail-archive.com/prometheus-announce@googlegroups.com/))
+    -- low-traffic list for announcements like new releases.
+-   [prometheus-users](https://groups.google.com/forum/#!forum/prometheus-users)
+    ([mirror](https://www.mail-archive.com/prometheus-users@googlegroups.com/))
+    -- for discussions around Prometheus usage and community support.
+    Announcements are *not* generally mirrored from
+    [prometheus-announce](https://groups.google.com/forum/#!forum/prometheus-announce).
+
+## Discourse forum
+
+Web-based discussion forum at [discuss.prometheus.io](https://discuss.prometheus.io/) hosted by
+[Discourse](https://www.discourse.org/).
+
+## Calendar for public events
+
+We have a public calendar for events, which you can use to join us.
+
+If you just want to get an overview, simply use our [web view in your browser\'s time zone](https://calendar.google.com/calendar/u/0/embed?src=prometheus.io_bdf9qgm081nrd0fe32g3olsld0%40group.calendar.google.com).
```

Comment:
There's a lot of instances of spurious backslashes like here: `\'`

Reply:
Yea, I used pandoc to convert the html. Will clean them up.